backtesting/backtesting.py

```diff
@@ -984,7 +984,8 @@ def __init__(self,
                  margin: float = 1.,
                  trade_on_close=False,
                  hedging=False,
-                 exclusive_orders=False
+                 exclusive_orders=False,
+                 close_all_at_end=True
```

Comment:
The type and amount of feedback following https://github.com/kernc/backtesting.py/issues/168 makes me think we should flip the default back to False.

Reply:
Missing a descriptive docstring below.

Reply:
Done.